Makefile

```diff
@@ -60,6 +60,7 @@ packaging/debian/changelog: packaging/debian/changelog.in
 
 $(WEBPACK_TEST): $(NODE_MODULES_TEST) $(LIB_TEST) $(shell find src/ -type f) package.json webpack.config.js
 	NODE_ENV=$(NODE_ENV) node_modules/.bin/webpack
+	[ -f dist/index.js.LICENSE.txt.gz ] || gzip </dev/null >dist/index.js.LICENSE.txt.gz
```

Comment:
I liked your idea about telling terser to only do the LICENSE bit, but not minification. Is that too difficult? This is oh-kay, but as it's so brittle and error prone I'd then really like to see some integration test that the file shipped by the RPM/deb is actually valid.

Reply:
Technically this should only run when NODE_ENV == development, so we can make the Makefile conditional, it's a bit of a hack. But then we don't need to test if the license is empty as it will not run.